instrumentation/aws_sdk/lib/opentelemetry/instrumentation/aws_sdk/instrumentation.rb

```diff
@@ -65,6 +69,15 @@ def gem_version
 
         private
 
+        def resolve_config(config)
```

Comment:
How are we providing backward compatability for the deprecated option?

Is it not supposed to work at all?